integration-test/utils/datafactory/schemas/codeofconduct.schema.ts

```diff
@@ -0,0 +1,41 @@
+import { pageSchema } from './page.schema';
+import { linkSchema } from './link.schema';
+
+export const codeofconductSchema = {
+  $ref: '#/definitions/pageSchema',
+  definitions: {
+    codeofconductSchema: {
+      type: 'object',
+      properties: {
+        id: {
+          type: 'string',
+          minLength: 1,
+        },
+        page: {
+          page: { ...pageSchema.definitions.pageSchema },
+        },
+        items: {
+          SectionString: {
+            properties: {
```

Comment:
Could you please check if you have the same code format configuration into your idea, looks a bit different the spacing in some attributes